test_unstructured_ingest/expected-structured-output/Sharepoint/Shared Documents/ideas-page.html.json

```diff
@@ -11,7 +11,95 @@
           "site_url": "https://unstructuredio.sharepoint.com"
         },
         "date_created": "2023-06-16T05:04:47",
-        "date_modified": "2023-06-16T05:04:47"
+        "date_modified": "2023-06-16T05:04:47",
+        "permissions_data": [
+          {
+            "id": "Q29tbXVuaWNhdGlvbiBzaXRlIE93bmVycw",
+            "roles": [
+              "owner"
+            ],
+            "shareId": "Q29tbXVuaWNhdGlvbiBzaXRlIE93bmVycw",
+            "grantedToV2": {
+              "siteGroup": {
+                "displayName": "Communication site Owners",
+                "id": "3",
+                "loginName": "Communication site Owners"
+              }
+            },
+            "grantedTo": {
+              "user": {
+                "displayName": "Communication site Owners"
+              }
+            },
+            "inheritedFrom": {}
+          },
+          {
+            "id": "Q29tbXVuaWNhdGlvbiBzaXRlIFZpc2l0b3Jz",
+            "roles": [
+              "read"
+            ],
+            "shareId": "Q29tbXVuaWNhdGlvbiBzaXRlIFZpc2l0b3Jz",
+            "grantedToV2": {
+              "siteGroup": {
+                "displayName": "Communication site Visitors",
+                "id": "4",
+                "loginName": "Communication site Visitors"
+              }
+            },
+            "grantedTo": {
```

Comment:
okay for now. at some point when we have >1 instance of this, we'll want to define our own schema for permissions.

Reply:
If users would like to interact with the source of the permission data again based on the ingested state of the permission data (ie, to update the permissions), they might need the original schema again. Considering that, if we want our own schema we can write parsers two ways: original->unstructured & unstructured->original

Reply:
yep, for sure.